tools/python/generate_plist.py

```diff
@@ -0,0 +1,41 @@
+import argparse
+import os
+
+plist_file_content = """
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>method</key>
+    <string>development</string>
+    <key>teamID</key>
+    <string>{team_id}</string>
+    <key>provisioningProfiles</key>
+    <dict>
+        <key>ai.onnxruntime.tests.ios-package-test</key>
```

Comment:
this is a pretty specific plist file, right? maybe put the script together with the other ios package test files? and consider a more specific name like generate_export_options_plist.py

Reply:
hmm renamed to generate_ipa_export_options_plist.py and moved into onnxruntime/test/platform/apple